.dockerignore

```diff
@@ -2,5 +2,6 @@
 
 !LICENSE
 !VERSION
+!fastentrypoints.py
```

Comment:
I usually take a different approach when building the container image that doesn't need the use of dockerignore files, but I believe we reviewed this at the early stages, and it was kind of the, probably, only way to move forward

Reply:
Exactly - this just follows the explicit convention of ignore files to ensure we only copy the required files during the build.